src/overlays/actors/ovl_Eff_Zoraband/z_eff_zoraband.c

```diff
@@ -30,16 +30,71 @@ const ActorInit Eff_Zoraband_InitVars = {
     (ActorFunc)EffZoraband_Draw,
 };
 
-#endif
+void EffZoraband_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EffZoraband* this = THIS;
 
-extern UNK_TYPE D_06000F38;
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = EffZoraband_Mikau_FadeOut;
+    this->alpha = 0;
+    this->actor.shape.rot.z = 0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Zoraband/EffZoraband_Init.s")
+void EffZoraband_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Zoraband/EffZoraband_Destroy.s")
+void EffZoraband_Mikau_FadeOut(EffZoraband* this, GlobalContext* globalCtx) {
```

Comment:
We don't usually include extra underscores unless a large subsystem is involved (see DmStk for an example).